packages/valory/skills/transaction_settlement_abci/models.py

```diff
@@ -37,6 +37,7 @@
 )
 
 
+_MINIMUM_VALIDATE_TIMEOUT = 300  # 5 minutes
```

Comment:
Hm that’s a bit aggressive? What about tests and test chains and faster chains?

Reply:
That's just the timeout, if it finishes earlier it should be fine. But I see your point, as there are cases where it is not necessary to wait for this long to assume that something has gone wrong. Based on the default value we set `1205`, I don't this is too big.